cpu/cpupower_monitor.py

```diff
@@ -41,7 +41,14 @@ def setUp(self):
             if 'Available idle states: ' in line:
                 self.states_list = (line.split('Available idle states: ')[-1]).split()
                 break
-        self.log.info("Idle states on the system are: ", self.states_list)
+        self.log.info("Idle states on the system are: %s", self.states_list)
```

Comment:
as using `%s` and `,` will not make substitution , either go with older way if still want to change please use this way 

`self.log.info("Idle states on the system are: %s" %self.states_list)`

as looking this opportunity i recommend to use string.format() using that we can get rid of all these issue

Reply:
Done.